bundles/org.openhab.binding.bluetooth.airthings/src/test/java/org/openhab/binding/bluetooth/airthings/AirthingsParserTest.java

```diff
@@ -61,6 +63,19 @@ public void testParsingPlus() throws AirthingsParserException {
         assertEquals(122, result.get(AirthingsDataParser.RADON_SHORT_TERM_AVG));
     }
 
+    @Test
+    public void testParsingWaveRadon() throws AirthingsParserException {
+        // Testdata from
+        // https://github.com/Airthings/airthings-ble/blob/9d255808fa3add6d504649e40c8548ffcd356909/tests/test_wave_plus.py#L47
+        byte[] data = HexFormat.of().parseHex("013860f009001100a709ffffffffffff0000ffff");
```

Comment:
Just to be consistent, would it make sense to use just an int array like it's used in other test?

Reply:
IMHO thus just shows the next dev it is easy to add testdata using data from the official AirThings repo that contains several other tests.